assets/base.css

```diff
@@ -3418,4 +3418,30 @@ details-disclosure > details {
     --easter-egg: none;
     --sparkle: url('./sparkle.gif');
   }
+
+  .animate--hover-raise .card-wrapper .card--card,
+  .animate--hover-raise .card-wrapper .card--standard .card__inner,
+  .animate--hover-raise .button:not(.button--tertiary),
+  .animate--hover-raise .shopify-challenge__button,
+  .animate--hover-raise .customer button,
+  .animate--hover-raise .shopify-payment-button__button,
+  .animate--hover-raise .deferred-media__poster-button {
+    transition: transform var(--duration-long) cubic-bezier(0,0,.35,1);
```

Comment:
If we're using the easing in multiple places we should put it in a variable, e.g. `--ease-out-slow`

Reply:
Done in https://github.com/Shopify/dawn/pull/2575/commits/ca9883a554ab9b087cadfbd8b19ffc181b70505f.